packages/vite-plugin/tests/out/vite-react-fast-refresh/__snapshots__/serve.test.ts.snap

```diff
@@ -170,15 +170,42 @@ export default App;
 var _c;
 $RefreshReg$(_c, \\"App\\");
 if (import.meta.hot) {
+  let isReactRefreshBoundary = function(mod) {
```

Comment:
some updated module? How is this different?

Reply:
good catch - turns out I had an outdated version of pnpm which couldn't handle the pnpm-lock.yaml file format, which resulted in pnpm ignoring it and ultimately in mismatched dependencies in my local setup, which caused unrelated test runs to diverge like this. Sorry for the confusion, should be fixed with latest force-push